otx/algorithms/classification/tasks/inference.py

```diff
@@ -319,14 +322,19 @@ def _init_recipe(self):
     # error log : ValueError: Unexpected type of 'data_loader' parameter
     def _init_model_cfg(self):
         base_dir = os.path.abspath(os.path.dirname(self.template_file_path))
+
+        model_base_name = "model"
+        if self._hyperparams.learning_parameters.enable_supcon:
+            model_base_name = "model_supcon"
+
         if self._multilabel:
-            cfg_path = os.path.join(base_dir, "model_multilabel.py")
+            cfg_path = os.path.join(base_dir, model_base_name + "_multilabel.py")
```

Comment:
Is there model_supcon_multilabel.py in template dir?

Reply:
There will be soon, once I push the changes in the MPA.
I think this PR should be on hold until the MPA changes are merged.
Then I will make another PR in the MPA adding support for multilabel and hierarchical classification.

Or, if you prefer, I can remove this and make another PR here later.

Reply:
Yes, let's make another PR for expand SupCon to multilabel and hierarchical classification